tests/e2e/config/jest.setup.js

```diff
@@ -108,104 +102,12 @@ function removePageEvents() {
 	} );
 }
 
-/**
- * Adds a page event handler to emit uncaught exception to process if one of
- * the observed console logging types is encountered.
- */
-function observeConsoleLogging() {
```

Comment:
Now that you deleted this, it means we won't catch console errors right? is this something that we want? couldn't we just skip `warning` type?

Reply:
@senadir You make a good point here. I've allowed console.error to surface, but only for debugging reasons. It's good to see them if they appear in the logs. The old comment mentioned that console.error breaks the tests, but that is no longer the case.